common/predictive-text/polyfills/symbol-es6.min.js

```diff
@@ -0,0 +1,2 @@
+// From https://rousan.io/symbol-es6/, lightly modified to be worker-compatible for Android Lollipop.
```

Comment:
What kind of modifications did you have to do to be "worker-compatible for Android Lollipop"?
in the event you need to re-create this.

Reply:
`typeof window?window:(self?self:global)`

There originally was no `self`-check; the polyfill assumed it was either in the `window` or that `global` would exist.  Naturally, being an old Android API, neither was true within the WebWorker.

Reply:
That was the only issue.